### DISCUSSION TITLE
Call a Power Virtual Agent Topic from a Composer Dialog

### DISCUSSION THREAD

**Discussion by marchantrr** [microsoft/BotFramework-Composer#6018] [Q&A]
I've been using the Bot Framework Composer integration with Power Virtual Agent and really like some of the additional capability that Composer provides.  Can anyone provide suggestions on how I can redirect to a PVA topic from a Composer Dialog?

From PVA topics, it's easy to redirect to Composer Dialogs or other PVA topics.

From Composer dialogs, I can redirect to other Composer dialogs but don't see a way of redirecting to PVA topics.

Any ideas?

**Comment by iMicknl** [comment]
@marchantrr I had the same question, which I logged here as a feature request: https://github.com/microsoft/BotFramework-Composer/issues/6047.

The answer there is that it will be supported soon;
> hello @iMicknl - this is on our short-term roadmap as new functionality as part of the integration. You should see this soon in an upcoming release
